Tests/NIOTests/TestUtils.swift

```diff
@@ -127,9 +127,7 @@ var temporaryDirectory: String {
         // for UNIX Domain Socket paths (which are limited to 103 bytes).
         return "/tmp"
 #else
-#if os(Android)
-        return "/data/local/tmp"
```

Comment:
`FileManager.default.temporaryDirectory.path` below handles this just fine, using the Termux `TMPDIR` in that app and this path otherwise.